a4_APIweatherApp.py

```diff
@@ -0,0 +1,90 @@
+'''
+This program is inteded to use the Open Meteo API
+and output an API and Mock plot of temperatures in the next 24 hours
+'''
+
+from abc import ABC, abstractmethod
+import requests
+import matplotlib.pyplot as plt
+
+class Service(ABC):
+    @abstractmethod
+    def getting(self):
+        pass
+
+class mockService(Service):
```

Comment:
Keep formatting of your class names standard. This should be `MockService`. Check these naming conventions, and try to use them for all python code you write:

https://peps.python.org/pep-0008/#naming-conventions